summertime/model/base_model.py

```diff
@@ -86,6 +86,16 @@ def generate_basic_description(cls) -> str:
 
         return basic_description
 
+    # TODO nick: implement this function eventually!
```

Comment:
should this be in the `base_model.py` or the `multingual_model`?

Reply:
Oh, I see that you are adding the function of returning "english" for non-multilingual models. Okay, then this is good.